src/components/Hero/Hero.module.css

```
@@ -3,45 +3,88 @@
 }
 
 .content {
+  position: relative;
+  z-index: 2;
   display: flex;
   flex-direction: column;
   align-items: center;
   text-align: center;
   gap: var(--mantine-spacing-xl);
+  max-width: 1000px;
+  background-color: rgba(9, 9, 11, 0.9);
```

Stylelint (17.7.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)